src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -10308,7 +10308,7 @@ void sofia_handle_sip_i_reinvite(switch_core_session_t *session,
 				tech_pvt->mparams.last_sdp_str = tech_pvt->mparams.prev_sdp_str;
 			}
 		}
-
+		sofia_glue_set_extra_headers(session, sip, SOFIA_SIP_HEADER_PREFIX);
```

Comment:
Looks good to me except that we need a new line after an if here.

Reply:
@andywolk I added an extra blank line after the closing bracket, is that what you mean? not really sure.